test/test_c10d.py

```diff
@@ -136,6 +136,12 @@ def broadcast(xs, rootRank, rootTensor):
                 self.assertEqual(torch.Tensor([i * self.size + j]), xs[0])
                 self.assertEqual(torch.Tensor([i * self.size + j]), xs[1])
 
+        # Test overloaded convenience function
+        x = torch.Tensor([self.rank + 1.0])
+        work = pg.broadcast(x, root=0)
+        work.wait()
+        self.assertEqual(torch.Tensor([1.0]), x)
```

Comment:
Actually another improvement we could make (to make sure the nicer API is tested as thoroughly as the main one) would be to wrap the `for i` loop above in `for broadcast in [broadcast_using_options, broadcast_sugared]:` where `broadcast_*` functions are written like the one above